### ISSUE TITLE
Clue Scroll Helper

### ISSUE THREAD

**Issue by AlexZabe** [runelite/runelite#2746]
Can you make the world map open when you read a cord clue.

**Comment by Infinitay** [comment]
I don't think we would because that means we'd have to fake input which we're against.

**Comment by ShaunDreclin** [comment]
Separate options for auto-opening the map and centering the map on the location would be nice, I'd only want the latter myself
